addons/mail/static/src/webclient/commands/command_category.js

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
Odoo module ?

Reply:
having a webclient/ folder in mail sounds weird. anyway, this should be reviewed by the discuss team I guess @seb-odoo

Reply:
Is it ready for us to be reviewed, or there are still changes to be applied?

Reply:
The guideline when overriding (or in this case registering) was (still is?) to use the same structure as the original code, so it looks ok to me.

I'm not against changing it either, but what better alternative do we have?

Reply:
> Is it ready for us to be reviewed, or there are still changes to be applied?

Yes, it s ready.